tmd/Tree/methods.py

```diff
@@ -178,21 +189,36 @@ def get_point_path_distances(self):
     path_lengths = np.append(0, copy.deepcopy(seg_len))
     children = get_children(self)
 
-    for i in children.keys():
-        path_lengths[children[i]] = path_lengths[children[i]] + path_lengths[i]
+    for i, child in children.items():
+        path_lengths[child] = path_lengths[child] + path_lengths[i]
 
     return path_lengths
 
 
+def get_point_trunk_length(self):
+    '''Tree method to get the trunk (first section length).
+    '''
+    ways, end = self.get_sections_only_points()
```

Comment:
Is this the right function or `self.get_sections_2()` ?

Reply:
Ah I don't know, I didn't see the `get_sections_2` function, maybe it's better. I will check that.

Reply:
This is the right function for what we want to do here :)

Reply:
ok, I am not sure what is the difference between these two methods. Their docstrings are identical yet they return different values.

Reply:
Indeed :)
From the tests I made, `get_sections_only_points` returns the index of the first and last points of a section while `get_sections_2` returns the index of the parent of the first point and the index of the last point of a section.
I can update the docstrings to make it more clear.